### PR TITLE
[BISERVER-13065] Scheduled task with deleted/renamed source report

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon please review

In this pull request:
 - created a JavascriptObject (JsPermissionsList) to parse the response from an endpoint called in the SchedulePanel.java (from pentaho-platform)
 - Added two new message properties, that represent the plural of, two existing properties.

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 20s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **315**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/gwt-widgets/job/PR-761/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **315**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/gwt-widgets/job/PR-761/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/resources/org/pentaho/mantle/public/messages/mantleMessages.properties [hunk 1]
```diff
@@ -89,6 +89,7 @@ deleteConfirmSchedles=Are you sure you want to delete {0} schedule(s)?
 deleteQuestion=Are you sure you want to permanently delete this {0}? <br>You can't undo this action.
 deleteAllQuestion=Are you sure you want to permanently delete all items in the Trash?<br>You can't undo this action.
 editScheduleResourceDoesNotExist=The source file cannot run or be edited because it may have been moved, deleted or permissions could have changed.  Do you want to delete this schedule?
+editScheduleResourceDoesNotExistMultiple=The source files cannot run or be edited because they may have been moved, deleted or permissions could have changed.  Do you want to delete this schedules?
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/mantle/public/messages/mantleMessages.properties]: `this schedules` -> `these schedules`

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 16s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **315**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/gwt-widgets/job/PR-761/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**